doc/plugin_server_nodeattestor_k8s_psat.md

```diff
@@ -75,4 +75,4 @@ This plugin generates the following selectors:
 The node and pod selectors are only provided for label keys in the `allowed_node_label_keys` and `allowed_pod_label_keys` configurables.
 
 
-A full example of this attestor is provided [here](../examples/k8s/simple_psat/README.md).
+A full example of this attestor is provided [here](https://github.com/spiffe/spire-examples/tree/54dbcbc8c378bb11c5040eb261ec51bc82f8a79b/examples/k8s/simple_psat).
```

Comment:
Any particular reason you're linking to a branch rather than master? Would a link to /spiffe/spire-examples/tree/master/examples/k8s/simple_psat be more appropriate?

Reply:
@ajessup I froze the tree-ish so it would be resistant to change (i.e. that url is not likely to ever change unless we mutate history), happy to just point to `/spiffe/spire-examples/tree/master/examples/k8s/simple_psat` instead too LMK

Reply:
Hmm. My default position would be to link to master since that should remain valid and remain up to date even as content here evolves, but I don't hold a strong opinion here TBH. @evan2645 or @azdagron do you have a preference?

Reply:
I feel we should link to master. Thank you.